src/jit/compiler.h

```diff
@@ -3861,13 +3861,26 @@ public :
     // Convert a BYTE which represents the VM's CorInfoGCtype to the JIT's var_types
     var_types   getJitGCType(BYTE gcType);
 
-    // Get the "primitive" type, if any, that is used to pass or return
-    // values of the given struct type.
-    var_types    argOrReturnTypeForStruct(CORINFO_CLASS_HANDLE clsHnd, bool forReturn);
+    enum structPassingKind { SPK_Unknown,             // Invalid value, never returned
+                             SPK_PrimitiveType,       // The struct is passed/returned using a primitive type
+                             SPK_ByValue,             // The struct is passed/returned by value, usually in multiple registers
```

Comment:
Yes I will add to the comment. This method is classifing what should be done, it doesn't actually implement the passing or returning of structure.